peachjam/templates/peachjam/layouts/base.html

```diff
@@ -40,7 +40,7 @@
       {% ifequal request.path '/articles/' %}
         <link rel="alternate"
               type="application/atom+xml"
-              title="Articles RSS Feed"
+              title="AfricanLII Articles RSS Feed"
```

Comment:
This is in `peachjam`, not AfricanLII, so you can't hardcode this. Use the APP_NAME from settings.py instead.